core/src/main/java/nl/nn/adapterframework/soap/SoapWrapperPipe.java

```diff
@@ -208,18 +207,16 @@ public PipeRunResult doPipe(Message message, PipeLineSession session) throws Pip
 				if (outputNamespaceTp != null) {
 					payload = new Message(outputNamespaceTp.transform(payload.asSource()));
 				}
-				Map<String,Object> parameterValues = null;
+				Map<String, Object> parameterValues = null;
 				if (!getParameterList().isEmpty() && (soapHeaderTp != null || soapBodyTp != null)) {
 					parameterValues = getParameterList().getValues(payload, session).getValueMap();
 				}
-				String soapHeader = null;
+				String soapHeader;
 				if (soapHeaderTp != null) {
 					payload.preserve();
 					soapHeader = soapHeaderTp.transform(payload, parameterValues);
 				} else {
-					if (StringUtils.isNotEmpty(getSoapHeaderSessionKey())) {
```

Comment:
Is het handig deze if weg te halen? De key kan leeg zijn?

Reply:
Goed punt, hij wordt idd gevuld als hij null is, door de configure. Zal het nog effe aanpassen.

Reply:
Oke, ik had deze if verwijderd nav een vraag van Tim. Nu ik een vraag van Niels krijg, check ik het nogmaals en zie dat hij alleen leeg is bij de wrap... tja. Ik zet de if terug. Vraag me af of dit soort verwijderingen überhaupt zinvol zijn om te overwegen, aangezien het ook niets oplevert als je zo'n check verwijderd.